tvb_epilepsy/service/model_inversion/statistical_models_builders.py

```diff
@@ -381,11 +409,17 @@ def generate_model(self):
         tic = time.time()
         self.logger.info("Generating model by " + self.__class__.__name__ + "...")
         parameters = self.generate_parameters()
-        model = SDEStatisticalModel(self.model_name, self.number_of_regions, self.xmode, self.priors_mode,
+        model = SDEStatisticalModel(self.name, self.number_of_regions, self.xmode, self.priors_mode,
                                     parameters, self.model_config, self.observation_model,
-                                    self.sigma_x, self.sigma_init, self.sigma, self.scale, self.offset,
+                                    self.sigma_x, self.sigma_init, self.sigma, self.scale, self.offset, self.epsilon,
                                     self.number_of_signals, self.time_length, self.dt, self.active_regions,
                                     self.sde_mode)
         self.logger.info(self.__class__.__name__  + " took " +
                          str(time.time() - tic) + ' sec for model generation')
-        return model
\ No newline at end of file
+        return model
+
+
+class EpileptorStatisticalModelBuilders(Enum):
+    STATISTICAL_MODEL_BUILDER = StatisticalModelBuilder()
+    ODESTATISTICAL_MODEL_BUILDER = ODEStatisticalModelBuilder()
+    SDESTATISTICAL_MODEL_BUILDER = SDEStatisticalModelBuilder()
```

Comment:
These will go away, in the next merge (from branch review), right?

Reply:
Yes, to all of the above.